### ISSUE TITLE
Progress bar for file copy/move [feature request]

### ISSUE THREAD

**Issue by nexus186** [nextcloud/server#11322]
When copying/moving large files, there is no progress bar shown other than a spinner (which goes away if you reload the page). When you have to move several folders sequentially, sometimes you forget and move the same folder (which results in an error).

It should at least display some kind of progress bar and a notification after the operation is complete. A drop down menu in the top navigation bar that displays an overlay on mouseover which lists each transfer with a progress bar would be nice.

**Comment by szaimen** [comment]
I think this is fixed in newer NC versions -> closing.

**Comment by J0WI** [comment]
> I think this is fixed in newer NC versions -> closing.

I don't think so. As of Nextcloud 21 there is no web UI for copy/move progress.

**Comment by anolting** [comment]
Hi @szaimen 
does the team has made any progress reviewing this feature request? I'd like also to see this feature as it's extremely annoying to wait for the spinner to disappear and not to know where the server is during larger copy operations. I'm talking about file amounts up 150k and sizes up to 500GB.
Best Alex

**Comment by joshtrichards** [comment]
Related: #41790

**Comment by george-andrei** [comment]
Hello, I am interested in this feature as well, and I see no progress on the related issue #41790
